courseware/factories.py

```diff
@@ -18,4 +21,8 @@ class Meta:
         model = OpenEdxApiAuth
 
     class Params:
-        expired = Trait(access_token_expires_on=Faker("past_datetime", tzinfo=pytz.utc))
+        expired = Trait(
+            access_token_expires_on=LazyAttribute(
+                lambda _: now_in_utc() - timedelta(days=1)
+            )
+        )
```

Comment:
Was there something that `Faker("past_datetime")` wasn't giving us?

Reply:
`freezegun` doesn't seem to affect certain `faker` generators, I spent a few hours trying to make it work but gave up and went with this instead